resources/js/utils/beatmapset-discussion-helper.ts

```diff
@@ -36,6 +49,26 @@ interface PropsFromHrefValue {
   target?: '_blank';
 }
 
+interface UrlOptions {
```

Comment:
I feel this and DiscussionPageUrlParams should have more similar name. Or maybe let the urlParse return this anyway.